dom/abort/event.any.js

```diff
@@ -64,4 +64,35 @@ test(t => {
   controller.abort();
 }, "the abort event should have the right properties");
 
+test(t => {
+  const controller = new AbortController();
+  const signal = controller.signal;
+
+  const reason = Error('hello');
+  controller.abort(reason);
+
+  assert_true(signal.aborted, "signal.aborted");
+  assert_equals(signal.reason, reason, "signal.reason");
+}, "AbortController abort(reason) should set signal.reason");
+
+test(t => {
+  const controller = new AbortController();
+  const signal = controller.signal;
+
+  controller.abort();
+
+  assert_true(signal.aborted, "signal.aborted");
+  assert_true(signal.reason instanceof AbortError, "signal.reason");
```

Comment:
This is wrong. It's an instance of DOMException, right? The name will be "AbortError". (Also wrong below.)